libiocage/cli/fetch.py

```diff
@@ -71,18 +70,15 @@
               help="Specify the files to fetch from the mirror. "
                    "(Deprecared: renamed to --file)")
 @click.option("--log-level", "-d", default=None)
-# @click.option("--auth", "-a", default=None, help="Authentication method for "
-#                                                 "HTTP fetching. Valid "
-#                                                 "values: basic, digest")
-# @click.option("--verify/--noverify", "-V/-NV", default=True,
-#               help="Enable or disable verifying SSL cert for HTTP fetching.")
-# def cli(url, files, release, update):
```

Comment:
Why are we removing current CLI flags?

Reply:
Because they were not implemented yet, so I removed it to clean up the file. Most probably https://github.com/iocage/libiocage/issues/43 will cause need to re-apply and use this option. I've updated the CLI criteria for signature checking, so that it's not forgotten.